src/parser/name_resolution.c

```diff
@@ -5125,7 +5151,29 @@ pt_dblink_table_get_column_defs (PARSER_CONTEXT * parser, PT_NODE * dblink, S_RE
 
   if (table_name)
     {
-      sprintf (sql_text, "SELECT * FROM %s", table_name);
+      int len = strlen (table_name);
+      char user_name[256];
+
+      /* make upper case for Oracle */
+      for (i = 0; i < len; i++)
+	{
+	  if (table_name[i] >= 'a' && table_name[i] <= 'z')
+	    {
+	      table_name[i] -= ('a' - 'A');
```

Comment:
In pt_check_dblink_column_alias(), intl_identifier_casecmp() was used.
Wouldn't it be a problem to switch only the English character area?

Instead of handling the strchr() in the code below, I think you can get the "find" variable here.